lang/RStudio/README.md

```diff
@@ -1,43 +1,56 @@
 # RStudio
 
-RStudio is an Integrated Development Environment (IDE) for the R language. By containerizing RStudio, a scientist can leverage the reproducibility and portability of the Singularity platform, allowing them to build data visualizations and analysis tools and ensure that they behave exactly the same in a new environment. 
+RStudio is an Integrated Development Environment (IDE) for the R language. By containerizing RStudio, a scientist can leverage the reproducibility and portability of the Singularity platform, allowing them to build data visualizations and analysis tools and ensure that they behave exactly the same in a new environment.
 
 The definiton file affords some flexibility, allowing you to run RStudio several different ways:
 
-## Instance:
+## Instance
+
  This will start up rstudio-server, on default port of 8787
 
-```$ singularity instance start rstudio.sif rs```
+```bash
+singularity instance start \
+  --bind run:/run,var-lib-rstudio-server:/var/lib/rstudio-server \
+  RStudio.sif rs --server-user=$(whoami) --www-address=0.0.0.0
+```
+
+## Run
 
-## Run:
  This will run rstudio IDE
 
-```$ singularity run rstudio.sif```
+```bash
+singularity run rstudio.sif
+```
+
+## Exec
 
-## Exec:
  Any program in the container, but specifically for running R directly.
 
-```$ singularity exec rstudio.sif R -f myfile.r```
+```bash
+singularity exec rstudio.sif R -f myfile.r
+```
 
 To build the image run:
 
-```$ sudo singularity build rstudio.sif RStudio.def```
+```bash
+sudo singularity build rstudio.sif RStudio.def
+```
 
 When you use run for the IDE, you may need to execute it like:
 
-
-```$ SINGULARITYENV_DISPLAY=${DISPLAY} \
-  singularity run rstudio.sif```
+```bash
```

Comment:
I would probably drop the stuff about `DISPLAY`. It will only not propagate if contain / compat / cleanenv options are used.... which they aren't here.